app/main/presenters/service_presenters.py

```diff
@@ -24,8 +24,8 @@ def _add_as_attribute_if_key_exists(self, key, service_data):
         if key[0] in service_data:
             setattr(self, key[1], service_data[key[0]])
 
-    def __init__(self, service_data, service_questions, lots_by_slug):
-        self.service_questions = service_questions
+    def __init__(self, service_data, builder, lots_by_slug):
+        self.summary_manifest = builder.summary(service_data)
```

Comment:
I'm fine with renaming it, but yeah, given the object is a ContentManifest, that's what my recommendation is based on. :)